planner/core/testdata/integration_suite_out.json

```diff
@@ -50,14 +50,14 @@
       {
         "SQL": "explain select * from t t1 left join t t2 on t1.a=t2.a where from_unixtime(t2.b);",
         "Plan": [
-          "HashJoin_8 9990.00 root  inner join, equal:[eq(test.t.a, test.t.a)]",
-          "├─Selection_13(Build) 7992.00 root  from_unixtime(cast(test.t.b))",
-          "│ └─TableReader_16 9990.00 root  data:Selection_15",
-          "│   └─Selection_15 9990.00 cop[tikv]  not(isnull(test.t.a))",
-          "│     └─TableFullScan_14 10000.00 cop[tikv] table:t2 keep order:false, stats:pseudo",
-          "└─TableReader_12(Probe) 9990.00 root  data:Selection_11",
-          "  └─Selection_11 9990.00 cop[tikv]  not(isnull(test.t.a))",
-          "    └─TableFullScan_10 10000.00 cop[tikv] table:t1 keep order:false, stats:pseudo"
+          "HashJoin_7 9990.00 root  inner join, equal:[eq(test.t.a, test.t.a)]",
+          "├─Selection_15(Build) 7992.00 root  from_unixtime(cast(test.t.b))",
+          "│ └─TableReader_14 7992.00 root  data:Selection_13",
+          "│   └─Selection_13 7992.00 cop[tikv]  not(isnull(test.t.a))",
```

Comment:
why the estimated output row count is changed from `9990.00` to `7992.00`?

Reply:
I think this is because `from_unixtime` is now pushed down by predicate pushdown rule, so when estimate output rowcount, the cop task thought it has two filter conditions(`is not null` and `from_unixtime`)